src/clj_test_containers/core.clj

```diff
@@ -93,7 +96,7 @@
 
 (defn init
   "Sets the properties for a testcontainer instance"
-  [{:keys [container exposed-ports env-vars command network network-aliases wait-for]}]
+  [{:keys [container exposed-ports env-vars command network network-aliases wait-for capture-logs?]}]
```

Comment:
Maybe handing in the condition in the beginning is a little to restrictive, e.g. if the users are only interested in the logs at a certain time. Maybe giving them more control, with functions to start and end the logging would be a little more flexible. What do you think?

Reply:
Yes, that is true. However, I imagine that since this a test container, and only really meant to be short-lived, then capturing-logs during definition/config of the container is acceptable (you probbably want to capture the logs during the test run, then grep it for something?) Not sure if you would want to setup that logging later...

Good point 'tho. Happy to take your direction on that one :-)

Reply:
The Testcontainers project suggests setting up a container at the beginning of the test phase and using it for a whole test run / suite if possible, so that setting would be used for many more tests than just one. It could be possible to pass the flag during creation and to provide functions to start and stop the logging. But I am more leaning towards providing a single point of configuration for this, as having both might be confusing to users